libs/libc/unistd/lib_getrlimit.c

```diff
@@ -55,5 +55,19 @@ int getrlimit(int resource, FAR struct rlimit *rlp)
   /* This is a dummy realization to make the compiler happy */
 
   memset(rlp, 0, sizeof(*rlp));
+
+  switch (resource)
+    {
+      case RLIMIT_NOFILE:
+        {
+          rlp->rlim_cur = 128;
+          rlp->rlim_max = 1024 * 1024; /* dummy */
```

Comment:
1024*1024->OPEN_MAX

Reply:
@xiaoxiang781216 
The value is determined from CONFIG_LIBC_OPEN_MAX, I have not been able to find this definition.
Thanks for pointing this out. I will correct it.